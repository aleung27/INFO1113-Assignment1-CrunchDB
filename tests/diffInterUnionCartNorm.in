SET a 2 4 5 1
SET b 1
SET c 2
SET d 0 9 8
DIFF a b c d
UNION a b c d
INTER a b c d
CARTPROD a b c d
DIFF d c b a
UNION d c b a
INTER d c b a
CARTPROD d c b a
POP b
DIFF a b c d
UNION a b c d
INTER a b c d
CARTPROD a b c d
