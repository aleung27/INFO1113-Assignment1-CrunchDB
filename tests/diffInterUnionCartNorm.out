> ok

> ok

> ok

> ok

> [0 4 5 8 9]

> [0 1 2 4 5 8 9]

> []

> [ [2 1 2 0] [2 1 2 9] [2 1 2 8] [4 1 2 0] [4 1 2 9] [4 1 2 8] [5 1 2 0] [5 1 2 9] [5 1 2 8] [1 1 2 0] [1 1 2 9] [1 1 2 8] ]

> [0 4 5 8 9]

> [0 1 2 4 5 8 9]

> []

> [ [0 2 1 2] [0 2 1 4] [0 2 1 5] [0 2 1 1] [9 2 1 2] [9 2 1 4] [9 2 1 5] [9 2 1 1] [8 2 1 2] [8 2 1 4] [8 2 1 5] [8 2 1 1] ]

> 1

> [0 1 4 5 8 9]

> [0 1 2 4 5 8 9]

> []

> [ [] ]

